README.md

```diff
@@ -58,11 +59,30 @@ case an implementation specific mechanism will be provided for registering the A
 The `io.opentracing.contrib.api.tracer.APIExtensionsTracer` provides a single constructor which is supplied
```

Comment:
Consistency. Why write the fully-qualified name here and `APIExtensionsManager` above?
I guess it's easier on the eyes to write the short name and maybe link to the java file?